pkg/odo/cli/experimental/experimental.go

```diff
@@ -0,0 +1,40 @@
+package experimental
+
+import (
+	"os"
+
+	"github.com/openshift/odo/pkg/config"
+	"github.com/openshift/odo/pkg/log"
+	"github.com/openshift/odo/pkg/preference"
+)
+
+// IsExperimentalModeEnabled checks if the experimental mode has been enabled
+// via env variable or via odo preferences and returns boolean value.
+// By default experimental mode is disabled
+func IsExperimentalModeEnabled() bool {
+
+	// Experimental mode can be set by:
+	//		- setting an env variable "ODO_EXPERIMENTAL=true" or
+	//		- setting odo preference using "odo preference set experimental true"
+
+	var (
+		experimentalPreference bool = false
+		experimentalEnv        bool = false
+	)
+
+	// Fetch odo preferences and check if experimental mode is set
+	cfg, err := preference.New()
+	if err != nil {
+		log.Errorf("failed to read odo preferences config. err: '%v'\n", err)
+	} else {
+		experimentalPreference = cfg.GetExperimental()
+	}
+
+	// Check "ODO_EXPERIMENTAL" env variable
+	experimentalEnvStr, _ := os.LookupEnv(config.OdoExperimentalEnv)
+	if experimentalEnvStr == "true" {
+		experimentalEnv = true
+	}
+
+	return (experimentalPreference || experimentalEnv)
```

Comment:
we don't need parentheses here

Reply:
Ok. It's just that the linter keeps complaining of this.

Reply:
The linter seems to have passed with this:
https://travis-ci.com/openshift/odo/jobs/285738324

Reply:
I mean the Goland editor's code style linter. But I will leave it up to you.